cfgov/unprocessed/apps/youth-employment-success/js/reducers/budget-reducer.js

```diff
@@ -0,0 +1,42 @@
+import keymirror from 'keymirror';
+
+const initialState = {
```

Comment:
Will this be the same state as in `BudgetFormView`? If so, does this perhaps indicate state should be spun out in its own module that can be shared? It may be too small to warrant spinning out.

Reply:
This state is going to replace the local state in `BudgetFormView`, since the tool will need that data is a few different places. I haven't actually gone through and wired up the reducer logic yet :/